azerothcore/azerothcore-wotlk
# fix(Core/Item): Fixed possible dupe with container items containing non un…

…ique non stackable items



## Changes Proposed:
-  Delete the correct item from item_loot_items - instead of a random one. If item container multiple items it would result in a dupe, refilling the item on startup.
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6939

## SOURCE:


## Tests Performed:

- This has been tested on a TC-based source code with many players online



## How to Test the Changes:


- Check the way items are stored, for example try to create a character, get some items, try to restart the server and make sure there is no regression and everything is normal with the stored items




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
